### ISSUE TITLE
Error installing cvs-fast-export

### ISSUE THREAD

**Issue by mojoTX** [Homebrew/homebrew-core#25884]
**Please note we will close your issue without comment if you delete, do not read or do not fill out the issue checklist below and provide ALL the requested information. If you repeatedly fail to use the issue template, we will block you from ever submitting issues to Homebrew again.**

- [X] are reporting a bug others will be able to reproduce and not asking a question. If you're not sure or want to ask a question do so on our Discourse: https://discourse.brew.sh
- [X] have a problem with `brew install` (or `upgrade`, `reinstall`) a single, official formula (not cask)? If it's a general `brew` problem please file this issue at Homebrew/brew: https://github.com/Homebrew/brew/issues/new. If it's a `brew cask` problem please file this issue at https://github.com/caskroom/homebrew-cask/issues/new. If it's a tap (e.g. Homebrew/homebrew-php) problem please file this issue at the tap.
- [X] ran `brew update` and can still reproduce the problem?
- [X] ran `brew doctor`, fixed all issues and can still reproduce the problem?
- [X] ran `brew gist-logs <formula>` (where `<formula>` is the name of the formula that failed) and included the output link?
_Yes, see this link:_  [https://gist.github.com/mojoTX/9491b04e23b145a038ff8d29e60bd963](https://gist.github.com/mojoTX/9491b04e23b145a038ff8d29e60bd963)
- [ ] if `brew gist-logs` didn't work: ran `brew config` and `brew doctor` and included their output with your issue?

To help us debug your issue please explain:
- What you were trying to do (and why)
_Install `cvs-fast-export`_ 
- What happened (include command output)
_I received an error:_
```
$ brew install cvs-fast-export
==> Downloading http://www.catb.org/~esr/cvs-fast-export/cvs-fast-export-1.43.tar.gz
Already downloaded: /Users/mjarvis/Library/Caches/Homebrew/cvs-fast-export-1.43.tar.gz
Error: wrong number of arguments (given 1, expected 0)
Please report this bug:
  https://docs.brew.sh/Troubleshooting
/usr/local/Homebrew/Library/Homebrew/formula.rb:1079:in `patch'
/usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/cvs-fast-export.rb:20:in `install'
/usr/local/Homebrew/Library/Homebrew/build.rb:140:in `block (2 levels) in install'
/usr/local/Homebrew/Library/Homebrew/formula.rb:1095:in `block in brew'
/usr/local/Homebrew/Library/Homebrew/formula.rb:1893:in `block (2 levels) in stage'
/usr/local/Homebrew/Library/Homebrew/utils.rb:558:in `with_env'
/usr/local/Homebrew/Library/Homebrew/formula.rb:1892:in `block in stage'
/usr/local/Homebrew/Library/Homebrew/resource.rb:119:in `block in unpack'
/usr/local/Homebrew/Library/Homebrew/extend/fileutils.rb:14:in `block in mktemp'
/usr/local/Homebrew/Library/Homebrew/extend/fileutils.rb:74:in `block in run'
/usr/local/Homebrew/Library/Homebrew/extend/fileutils.rb:74:in `chdir'
/usr/local/Homebrew/Library/Homebrew/extend/fileutils.rb:74:in `run'
/usr/local/Homebrew/Library/Homebrew/extend/fileutils.rb:13:in `mktemp'
/usr/local/Homebrew/Library/Homebrew/resource.rb:114:in `unpack'
/usr/local/Homebrew/Library/Homebrew/resource.rb:92:in `stage'
/usr/local/Homebrew/Library/Homebrew/formula.rb:1870:in `stage'
/usr/local/Homebrew/Library/Homebrew/formula.rb:1090:in `brew'
/usr/local/Homebrew/Library/Homebrew/build.rb:111:in `block in install'
/usr/local/Homebrew/Library/Homebrew/utils.rb:558:in `with_env'
/usr/local/Homebrew/Library/Homebrew/build.rb:108:in `install'
/usr/local/Homebrew/Library/Homebrew/build.rb:189:in `<main>'
```
- What you expected to happen
_I expected it to install_

- Step-by-step reproduction instructions (by running `brew install` commands)

```
brew install cvs-fast-export
```

# Formula additions or changes
To get formulae added or changed in Homebrew please file a [Pull Request](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)
We will close issues requesting formulae changes.

**Comment by fxcoudert** [comment]
```
$ brew info cvs-fast-export   
Error: No available formula with the name "cvs-fast-export" 
==> Searching for a previously deleted formula (in the last month)...
Error: No previously deleted formula found.
```

It seems you have a `cvs-fast-export` formula, but it does not come from us:

```
/usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/cvs-fast-export.rb:20:in `install'
```

At line 20 of that formula, the `patch` method is called incorrectly.
Closing as this is not a problem arising from Homebrew core.

**Comment by mojoTX** [comment]
Wow, I'm now embarrassed! Sorry about that! 

Now that I think about it, I think the last time I had to do a CVS-to-Git import, I looked for it in Homebrew first, and then I must have started creating the recipe myself. It was probably some time last summer. 

However, getting the actual project imported was more important than writing the recipe, so I just downloaded and installed by hand, and then never finished the recipe, and then forgot all about it (until today). 

I'll see if I can finish it up. I fixed the patch problem, but now I see other issues.
